### PR TITLE
feat(admin-ui): add feature to upload .jwt file 

### PR DESCRIPTION
**Pull request by mjatin-dev**
https://github.com/GluuFederation/flex/issues/874#issuecomment-1483224292

### CODE REVIEW

File: admin-ui/app/utils/UploadSSA.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  } = useDropzone({
+    onDrop: onDrop,
+    accept: {
+      'application/jwt': ['.jwt'],
+    },
+  })
+
+  const submitData = () => {
+    if(selectedFile){
+        dispatch(uploadNewSsaToken({licenseKey:jwt}))
```
Comment (duttarnab) [inline admin-ui/app/utils/UploadSSA.js:61]: `jwt` is `ssa`. The `licenseKey` name should be `ssa`.

File: admin-ui/app/redux/api/LicenseApi.js [hunk 2]
```diff
@@ -21,6 +21,22 @@ export default class LicenseApi {
       })
     })
   }
+  uploadSSAtoken = (data) => {
+    let option = {}
+    option['sSARequest'] = {ssa:data.licenseKey}
```
Comment (duttarnab) [inline admin-ui/app/redux/api/LicenseApi.js:26]: `jwt` is `ssa`. The `licenseKey` name should be `ssa`.

So,
 `option['sSARequest'] = {ssa:data.ssa}`

**Review by duttarnab — COMMENTED** [review]
See inline comments.

File: admin-ui/app/utils/UploadSSA.js (hunk 1 above)
Comment (mjatin-dev) [reply admin-ui/app/utils/UploadSSA.js]: Changed as suggested

File: admin-ui/app/redux/api/LicenseApi.js (hunk 2 above)
Comment (mjatin-dev) [reply admin-ui/app/redux/api/LicenseApi.js]: Change it to `option['sSARequest'] = data` as it is more suitable in this case.

**Pull request merged**